master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
```

Comment:
I'd say a Node resource (and you could link to the resource definition if you like).

There are various bits of node-related configuration (node specific low level config, workload endpoints, host endpoints, IP address assignments, IP CIDR affinity blocks).